app/packages/operators/src/usePanelEvent.ts

```diff
@@ -22,6 +22,10 @@ export default function usePanelEvent() {
     const options = args[0] as HandlerOptions;
     const { params, operator, prompt, currentPanelState } = options;
 
+    if (!operator) {
+      return console.warn("No operator provided for panel event.");
```

Comment:
Add more info? Or a trace… how would you figure out where this is occurring.

Reply:
Ohh, good catch. This is leftover and is unrelated. Let me clean this PR up to remove unrelated changes.